styles.css

```diff
@@ -0,0 +1,86 @@
+body {
+  background-color: white;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] You design stick to the Figma design , however make sure you solve this problem of  screen overflow , it will make your web page more responsive and professional :+1: 

[Screencast from 02-17-2023 01_53_30 PM.webm](https://user-images.githubusercontent.com/80458199/219647792-57eefad4-2e05-4cca-9489-6aab38f12cbc.webm)

Reply:
Hello @iambenkis, your feedback is very progressive for me, thank you for your kind suggestions. However, I may need some more clarity on this particular review. Can you please elaborate a little more on that? Looking forward to your feedback. Thanks!

Reply:
> * [ ]  You design stick to the Figma design , however make sure you solve this problem of  screen overflow , it will make your web page more responsive and professional 👍

@iambenkis Please consider my request that I have posted as a comment on this review 👍